client/src/components/form/EducationInfo.tsx

```
@@ -52,249 +40,118 @@ const departments = [
-      university: formData.university || {
-        faculty: "",
-        department: "",
-        session: "",
+function AcademicDetails({ section, formData, setFormData }: { section: "ssc" | "hsc", formData: EducationInfoData, setFormData: (data: EducationInfoData) => void }) {
+  const handleChange = (e: ChangeEvent<HTMLSelectElement | HTMLInputElement>, field: keyof AcademicRecord) => {
+    const { value } = e.target;
+    setFormData({
+      ...formData,
+      [section]: {
+        ...formData[section],
+        [field]: field === "passingYear" ? parseInt(value) : value,
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)